docs/getting-started/delta-lake-format.md

```diff
@@ -88,6 +88,7 @@ Finally, adapt the action definition for `join-departures-airports`:
 To run our data pipeline, first delete the data directory - otherwise DeltaLakeTableDataObject will fail because of existing files in different format.
 Then you can execute the usual *docker run* command for all feeds:
 
+    mkdir -f data
```

Comment:
What does `-f` do? Do you mean `-p` ?

Reply:
-f does not fail if the directory already exists. Works ok Linux and Windows.
I changed it cos the command failed when I already had a data directory I believe. Like this, you can now copy paste the whole 3 rows and it will work, regardless of the directories you have

Reply:
Ah OK, then I propose to use `-p`. It also doesn't fail if the directory already exists, and also creates parent directories if needed so you can skip the `mkdir -f data`.

Here's the explanation from the man page:
```
      -p, --parents
              no error if existing, make parent directories as needed
```

`-f` is not mentioned in the man page.

Reply:
actually you right: -p only fails when data and data/_metastore exist, which makes sense. I will keep this one as is